environment_setup/install_cubemx.sh

```diff
@@ -3,9 +3,10 @@
 # Any failure should terminate this script
 set -e
 
-# download version 5.4.0 (change the following two variables to change versions)
-CUBE_VERSION="5.4.0"
-CUBE_LINK="https://www.st.com/content/ccc/resource/technical/software/sw_development_suite/group0/5f/d0/cf/79/10/fb/4e/7e/STM32CubeMX_v5-4-0/files/stm32cubemx_v5.4.0.zip/jcr:content/translations/en.stm32cubemx_v5.4.0.zip"
+# download version 5.6.1 (change the following three variables to change versions)
+CUBE_VERSION="5.6.1"
+CUBE_ZIP_FILENAME="en.stm32cubemx_v5-6-1.zip"
+CUBE_LINK="https://www.st.com/content/ccc/resource/technical/software/sw_development_suite/group0/01/9e/e3/0c/3d/ff/41/7b/stm32cubemx_v5-6-1/files/stm32cubemx_v5-6-1.zip/jcr:content/translations/en.stm32cubemx_v5-6-1.zip"
```

Comment:
is it worth explaining where/how we got this link? Just in case people have to do something with it in the future?

Reply:
added